jest.config.js

```diff
@@ -17,8 +17,7 @@ module.exports = {
   },
   transformIgnorePatterns: [`/node_modules/(?!${esModules})`],
   moduleNameMapper: {
-    '^.+\\.(css)$': 'identity-obj-proxy',
-    '^.+\\.(svg)$': 'identity-obj-proxy',
+    '^.+\\.(css|svg|png)$': 'identity-obj-proxy',
```

Comment:
A recent change in [`stripes-smart-components`](https://github.com/folio-org/stripes-smart-components/pull/1464/files) requires the addition of `png`. Otherwise tests will fail.